packages/launchpad/src/global/GlobalPage.spec.tsx

```diff
@@ -1,19 +1,21 @@
 import { defaultMessages } from '../locales/i18n'
 import GlobalPage from './GlobalPage.vue'
+import { GlobalPageFragmentDoc } from '../generated/graphql-test'
 
 const searchSelector = `input[placeholder="${defaultMessages.globalPage.searchPlaceholder}"`
 const emptyMessages = defaultMessages.globalPage.empty
+const testProject = 'test-project'
+const anotherTestProject = 'another-test-project'
+const testProjectPath = '/usr/local/dev/projects/test-project'
 
 describe('<GlobalPage />', { viewportHeight: 900, viewportWidth: 1200 }, () => {
-  beforeEach(() => {
-    cy.mount(() => (<div>
-      <GlobalPage />
-    </div>))
-  })
-
   // TODO: add gql so that we can mock out the fragment response
   describe.skip('without projects', () => {
     it('renders the empty state', () => {
+      cy.mount(() => (<div>
```

Comment:
Seems this entire test is skipped with `.skip`. I wonder if we just delete it? The other tests is this file seem to cover everything this one does anyway.

Reply:
If you don't have any projects it will show the drag and drop screen. I didn't see the skip after describe and removed that so we have this test still